crates/ra_ide_api/src/expand_macro.rs

```diff
@@ -0,0 +1,113 @@
+//! FIXME: write short doc here
```

Comment:
Let's try to avoid adding new modules with documentation. Something like this will work

> This modules implements "expand macro" functionality in the IDE